### PR TITLE
Adding notes for Jan and Feb 2019

### PR DESCRIPTION
**Pull request by sffc**

### CODE REVIEW

File: meetings/notes-2019-01-17.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+This is an initial agenda document; please add additional agenda items as appropriate through PRs to this repository.
```
Comment (littledan) [inline meetings/notes-2019-01-17.md]: Please delete this line from the notes

File: meetings/notes-2019-01-17.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+January 17th, 2019, 16:00 UTC
+
+Attendees: Nebojsa Ciric (NC), Frank Tang (FT),  Daniel Ehrenberg (DE), Shane Carr (SC), Felipe Balbontin (FB), Ben Michel (BM), Richard Gibson (RG), Mark Davis (MD), Zibi Braniecki (ZB), Sathya Gunasekaran (SG)
+
+Contact Daniel Ehrenberg (littledan@igalia.com) for the link to the Google Hangouts link.
```
Comment (littledan) [inline meetings/notes-2019-01-17.md]: Delete this line as well

File: meetings/notes-2019-02-21.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+        43. SL: But you're saving effort for managing all that data.
+        44. MM: They have to manage lots of other kinds of data as well.
+        45. DE: Maybe we could broaden the pool of web developers that we talk to about this API. We have one on this call, who said that it would be very useful, but we could talk to more to understand whether their needs match SL's or MM's estimate
+        46. MM: That sounds like a good idea.
+        47. SL: With the Japanese era change, there will be a big roll-out that will be hard to manage. We could centralize this.
+        48. MM: Is era name included in this proposal?
+        49. FT: Not here.
+        50. SL: Diale[ct? https://github.com/tc39/proposal-intl-displaynames/issues/20](https://github.com/tc39/proposal-intl-displaynames/issues/20) **nl-BE** - Flemish , not (Dutch in Belgium)
+        51. SL: I think we need an option for this, whether you want to see "Flemish" or "Dutch (Belgium)"
+        52. [Locale vs Language](https://github.com/tc39/proposal-intl-displaynames/issues/12) - opinions?
```
Comment (littledan) [inline meetings/notes-2019-02-21.md]: Delete agenda items that we did not discuss as well.

**Comment by sffc** [comment]
Feedback resolved.  Will squash and merge.

**Pull request merged**